src/tokenize.cpp

```diff
@@ -11,6 +11,7 @@
 #include "char_table.h"
 #include "prototypes.h"
 #include "chunk_list.h"
+#include <stdbool.h>
```

Comment:
@CDanU you are right the stdbool include is not necessary. I added it when I saw the linker error in the release build. I'll remove the unnecessary include.